File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Variations/Variation.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Variations;

class Variation  implements ILabel
{
    public $Details;//VariationList
    public $Products;//ProductList
    public $Packages;//PackageList
    public $ID;
    public $Name;
    public $Title;
    public $State;
    public $OrderNo;
    public $Description;
}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Variations/Variation.cs
using Ohtap.ECommerce.Domain.Models.Objects.Products;
using Ohtap.ECommerce.Domain.Models.Objects.VariationDetails;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.Variations
{
    public class Variation : VariationBasic, IVariation
    {
        public virtual Packages.IPackageDataList Packages { get; set; }

        public virtual IProductDataList Products { get; set; }
        IVariationDetailDataList IVariation.Details { get; set; }
    }
}
